fudy/NoteScreen/Component/FramelessWindow.qml

```diff
@@ -100,9 +103,9 @@ ApplicationWindow {
 		id: mouseArea
 
 		anchors.fill: parent
-
+		hoverEnabled: true
 		acceptedButtons: Qt.LeftButton
-		pressAndHoldInterval: internal.interval
+		pressAndHoldInterval: 100
 		onPressAndHold: function(mouse) {
 			if (mouse.x >= parent.width - internal.margin || mouse.x <= internal.margin || mouse.y >= parent.height - internal.margin) {
```

Comment:
Long line

Reply:
![Screenshot 2024-07-24 005154](https://github.com/user-attachments/assets/476b102e-3087-483c-ba16-72b19faf45a2)
The red area is the zone that allows the mouse to modify the size of the Note window.

Reply:
I mean you should break it into 2 or 3 lines

Reply:
got it, thank you